tests/garage/torch/algos/test_maml.py

```diff
@@ -0,0 +1,97 @@
+"""Unit tests of MAML."""
+from functools import partial
+
+import pytest
+import torch
+
+from garage.envs import HalfCheetahDirEnv, normalize
+from garage.envs.base import GarageEnv
+from garage.np.baselines import LinearFeatureBaseline
+from garage.sampler import LocalSampler, WorkerFactory
+from garage.torch.algos import MAMLPPO
+from garage.torch.policies import GaussianMLPPolicy
+
+
+@pytest.fixture
+def set_params():  # noqa: D202
+    """Set parameters of a module."""
+
+    def _set_params(v, m):
```

Comment:
do these really need to be fixtures as opposed to just helper functions? pytest fixtures usually represent a resource, mocked class, or are used for some kind of metaprogramming,